src/EnergyPlus/RefrigeratedCase.cc

```diff
@@ -8634,8 +8634,9 @@ void SetupReportInput(EnergyPlusData &state)
                                         OutputProcessor::StoreType::Sum,
                                         rack.Name);
 
+                    // If CoilFlag is true and Location is Zone, GetRefrigerationInput ensures you must have a Heat Rejection Zone provided already
                     SetupZoneInternalGain(state,
-                                          RefrigCase(rack.CaseNum(1)).ActualZoneNum,
+                                          rack.HeatRejectionZoneNum,
```

Comment:
In this case that means NumCoils > 0, Get input routine will throw if you don't have a HeatRejectionZone, so safe to use